source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,65 @@
+.. _java-rs-get-started-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install Dependencies
+    
+      Ensure you have `JDK version 8 or later
+      <https://www.oracle.com/java/technologies/javase-downloads.html>`__ installed
+      in your development environment.
+
+      This guide shows you how to add the MongoDB Java driver dependencies using 
+      Maven or Gradle in an integrated development environment (IDE). We
+      recommend that you use an IDE such as IntelliJ IDEA or Eclipse IDE. An IDE
+      makes it more convenient to configure Maven or Gradle to build and run
+      your project.
+
+      If you are not using an IDE, see
+      `Building Maven <https://maven.apache.org/guides/development/guide-building-maven.html>`_
+      or
+      `Creating New Gradle Builds
+      <https://guides.gradle.org/creating-new-gradle-builds/>`_
+      for more information about how to set up your project.
+
+   .. step:: Install the {+driver-short+}
```

Comment:
Hi! I was wondering -- could we include a note similar to the note in the Java Sync docs [here](https://www.mongodb.com/docs/drivers/java/sync/current/quick-start/)? Basically saying that while we recommend users use a build tool to configure their dependencies, users _can_ download the driver and dependencies themselves from Maven [here](https://mvnrepository.com/artifact/org.mongodb/mongodb-driver-reactivestreams)?